modules/ROOT/pages/_partials/common-read-operation.adoc

```diff
@@ -1,9 +1,9 @@
-== Mime Types and Encoding
+== MIME Types and Encoding
 
-This connector tries to determine a file’s mime type (`mimeType`) based on its extension. However, when first-hand knowledge of the file’s content is necessary, you can force that `mimeType` to a particular value by using the `outputMimeType` parameter.
+Each connector tries to determine a file’s MIME type (`mimeType`) from its extension. However, if first-hand knowledge of the file’s content is necessary, you can force the MIME type to a different value with the `outputMimeType` parameter.
 
-The same process works for encoding. By default, the connector assumes that Mule runtime’s default encoding matches the one in the file. However, you can set the encoding through the `outputEncoding` parameter.
+The same process works for encoding. By default, the connector assumes that the Mule runtime’s default encoding matches the MIME type of the file. However, you can set the encoding through the `outputEncoding` parameter.
```

Comment:
"the Mule runtime’s" -> "the Mule runtime" ? or keep as was "Mule runtime's"

Reply:
Changed to:
By default, the connector assumes that the  default encoding in the Mule runtime matches the MIME type of the file.